gitlab/objects.py

```diff
@@ -787,6 +802,7 @@ class Project(GitlabObject):
     managers = [
         ('branches', ProjectBranchManager, [('project_id', 'id')]),
         ('commits', ProjectCommitManager, [('project_id', 'id')]),
+        ('statuses', ProjectCommitStatusManager, [('project_id', 'id')]),
```

Comment:
For consistency I'd call the attribute 'commitstatuses' instead of 'statuses' (to match the class name).